dev/com.ibm.ws.logging/src/com/ibm/ws/logging/source/LogSource.java

```diff
@@ -183,6 +183,7 @@ public GenericData parse(RoutedMessage routedMessage) {
         genData.setSourceType(sourceName);
         genData.setLogRecordLevel(logRecord.getLevel());
         genData.setLoggerName(logRecord.getLoggerName());
+        genData.setLogLevel(LogFormatUtils.mapLevelToRawType(logRecord));
```

Comment:
pair has already been added to genData above for mapLevelToRawType value.  should just use that rather than adding another field to genData.